### ISSUE TITLE
Crash to Desktop - Tesselating Block

### ISSUE THREAD

**Issue by Utildayael** [aidancbrady/Mekanism#3737]
Crash to desktop while working with Mekanism wind mills.

Short error:

```
Description: Tesselating block in world

java.lang.ClassCastException: net.minecraft.block.BlockAir cannot be cast to mekanism.client.render.ctm.ICTMBlock
	at mekanism.client.render.ctm.CTM.canConnect(CTM.java:320)
	at mekanism.client.render.ctm.CTM.isConnected(CTM.java:301)
	at mekanism.client.render.ctm.Dir.isConnected(Dir.java:81)
	at mekanism.client.render.ctm.CTM.buildConnectionMap(CTM.java:191)
	at mekanism.client.render.ctm.CTM.createSubmapIndices(CTM.java:154)
	at mekanism.client.render.ctm.CTMBlockRenderContext.<init>(CTMBlockRenderContext.java:26)
	at mekanism.common.block.BlockBasic.getExtendedState(BlockBasic.java:217)
	at net.minecraft.client.renderer.BlockRendererDispatcher.func_175018_a(BlockRendererDispatcher.java:80)
	at mods.betterfoliage.client.Hooks.renderWorldBlock(Hooks.kt:97)
	at net.minecraft.client.renderer.chunk.RenderChunk.func_178581_b(RenderChunk.java:199)
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.func_178474_a(SourceFile:100)
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:43)
	at java.lang.Thread.run(Unknown Source)
```

Full log on pastebin:

http://pastebin.com/NVx3LqQR

Mek 9.1.1
MC 1.10.2
Forge 2065

Thanks

**Comment by aidancbrady** [comment]
Fixed in development, thanks!

**Comment by Utildayael** [comment]
You're on a roll today!

**Comment by aidancbrady** [comment]
Just trying to take advantage of labor day!

**Comment by FernandoVpro** [comment]
Please can you fix this error for version 1.12.1 PS: I love your mod

**Comment by thommy101** [comment]
If you're having exactly the same crashlog, then theres something wrong, this code isn't in 1.12.1 anymore. 

If you have crashes and don't know if this exactly this issue, please open a new issue with your crashlog.
